template/internal/db/migrations/migrations.go

```diff
@@ -1,8 +1,8 @@
 package migrations
 
 import (
-	_ "embed"
+	"embed"
 )
 
-//go:embed migrations.sql
-var Migrations string
+//go:embed *.sql
+var MigrationDir embed.FS
```

Comment:
this actually has a lot of stutter which is against convention, 
since its in the migrations package we can call this vara Dir or just keep with migrations